compose-builder/.env

```diff
@@ -45,13 +45,13 @@ DEVICE_S7_VERSION=latest
 DEVICE_OPCUA_VERSION=latest
 DEVICE_CAN_VERSION=latest
 
-BAO_VERSION=2.1.1
+BAO_VERSION=2.1
 POSTGRES_VERSION=16.3-alpine3.20
 KUIPER_VERSION=v2.1.0-beta.4
```

Comment:
```suggestion
KUIPER_VERSION=v2.1
```

Reply:
there is no v2.1 docker tag

Reply:
Use v2.1-alpine?

Reply:
here is 2.1 and 2.1-alpine
https://hub.docker.com/layers/lfedge/ekuiper/2.1/images/sha256-6889ae083c71f6449912f5451765e1af518b6f1bdd7d96e2ec62c0546c7854ef

https://hub.docker.com/layers/lfedge/ekuiper/2.1-alpine/images/sha256-6d878b9b94726cf415e0330f13ceef9405a8288086c8ee235020f3933b4dcb21